ClemBot.Bot/bot/cogs/info_cog.py

```diff
@@ -0,0 +1,77 @@
+            user_info += '\n» **Created:** ' + user.created_at.strftime('%b %d %Y %I:%M:%S %p')
+        embed.add_field(name='**User ID:**', value=user_info)
+        embed.set_thumbnail(url=user.avatar_url)
+
+        #since member objects include guild ID's and information, we try to get a member object for the target user
+        member = ctx.guild.get_member(user.id)
+        #check to see if the target user is in the calling context's guild. If None then the user isn't in the calling server
+        if member:
+
+            # We don't want moderation info available for regular guild users to check on others
+            if (await self.bot.claims_check(ctx=ctx)): 
+                warnings = await self.bot.moderation_route.get_guild_warns_user(ctx.guild.id, user.id)
+                mutes = await self.bot.moderation_route.get_guild_mutes_user(ctx.guild.id, user.id)
+                infractions = await self.bot.moderation_route.get_guild_infractions_user(ctx.guild.id, user.id)
+                
+                moderation_info = f'» **Warnings:** {0 if warnings is None else len(warnings)} '
+                moderation_info += f'\n» **Mutes:** {0 if mutes is None else len(mutes)}'
+                moderation_info += f'\n» **Total Infractions:** {0 if infractions is None else len(infractions)}'
+                embed.add_field(name='**Moderation Info:**', value=moderation_info)
+
+            guild_info = ''
+            if (ctx.command.claims_check(claims)) or (ctx.author.id == user.id):
+                #prints the datetime in the format of <Months> <day> <year> <hours>:<minutes>:<seconds> <AM/PM>
+                guild_info = '» **Joined:** ' + member.joined_at.strftime('%b %d %Y %I:%M:%S %p') 
+            #will re-add message counts soon
+            # messages = await self.bot.message_route.get_message_count_range(ctx.guild.id, user.id)
+            # guild_info += f'\n» **Message count (last 30 days):** {await MessageRepository().get_user_message_count_range(member.id, ctx.guild.id, 30)}'
+            guild_info += f'\n» **Roles:** '
+            log.info(f'User has roles: {member.roles}')
+            #skipping the first index because it is just @everyone
+            for i in member.roles[1::]:
+                guild_info += f'{i.mention}\t'
+            guild_info += f'\n» **Highest role:** {member.top_role.mention}'
+            guild_info += '\n» **Nitro boost date:** ' + (member.premium_since.strftime('%b %d %Y %I:%M:%S %p') if member.premium_since is not None else 'Not boosting')
+            embed.add_field(name='**Guild Information:**', value=guild_info, inline=False)
+
+        embed.set_footer(text=f'{ctx.author.name}#{ctx.author.discriminator}', icon_url=ctx.author.avatar_url)
+        msg = await ctx.send(embed=embed)
+        await self.bot.messenger.publish(Events.on_set_deletable, msg=msg, author=ctx.author)
+        return
```

Comment:
This `return` is not needed.
```suggestion
```

Reply:
I prefer to keep it there to more easily see that it is the end of a void function